content/news/2021-09-13-darktable-3.6.1-released/index.md

```diff
@@ -0,0 +1,81 @@
+---
+author: Pascal Obry
+date: 2021-09-13 10:00:00+00:00
+layout: post
+title: "darktable 3.6.1 released"
+lede: vassiviere.jpg
+lede_author: <a href="http://photos.obry.net">Pascal Obry</a>
+tags:
+  - announcement
+  - darktable release
+---
+
+We're proud to announce the new feature release of darktable, 3.6.1!
+
+The github release is here: [https://github.com/darktable-org/darktable/releases/tag/release-3.6.1](https://github.com/darktable-org/darktable/releases/tag/release-3.6.1).
+
+As always, please don't use the autogenerated tarball provided by
+github, but only our tar.xz file. the checksums are:
+
+```
+$ sha256sum darktable-3.6.1.tar.xz
+a2bfc7c103b824945457a9bfed9e52f007fa1d030f9dbcb3ff0327851be42d14 darktable-3.6.1.tar.xz
+$ sha256sum darktable-3.6.1.5.dmg
+0fa0cd49971fdf76f690be095600250ad7777c2317f969c7135f035bc7b5f0ee darktable-3.6.1.5.dmg
+$ sha256sum darktable-3.6.1.exe
+34b4c1748ffe7ac66c3c79ad474b86a7aaa01ba296f8cd21929fe8024ebf3262 darktable-3.6.1-win64.exe
+```
+
+When updating from the currently stable 3.4.x series, please bear in
```

Comment:
Perhaps we shouldn't say "currently stable" here, since the "current" stable version is 3.6.0

Reply:
No this is fine. There is no issue from 3.6.0 to 3.6.1 as it does not change the db.

Reply:
I understand. I don't have a problem with the sentence as a whole, just remove "currently stable" because it implies that 3.4.x is the current version. So just say "When updating from the 3.4.x series, please bear in mind..."